giga_connectome/connectome.py

```diff
@@ -41,12 +46,12 @@ def build_size_roi(mask: np.ndarray, labels_roi: np.ndarray) -> np.ndarray:
 
 
 def calculate_intranetwork_correlation(
-    correlation_matrix: np.array,
-    masker_labels: np.array,
-    time_series_atlas: np.array,
-    group_mask: Union[str, Path, Nifti1Image],
-    atlas_image: Union[str, Path, Nifti1Image],
-) -> Tuple[np.ndarray, np.ndarray]:
+    correlation_matrix: np.ndarray[Any, Any],
```

Comment:
Is this how to declare the dimension of the numpy array? That's cool

Reply:
I need to investigate better how to use types with numpy, this is a bit of a brute force approach.